pkgs/development/libraries/libxml2/common.nix

```diff
@@ -87,9 +86,12 @@ stdenv'.mkDerivation (finalAttrs: {
   propagatedBuildInputs = [
     findXMLCatalogs
   ]
-  ++ lib.optionals (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isMinGW) [
-    libiconv
-  ]
+  ++
+    lib.optionals
+      (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isMinGW || stdenv.hostPlatform.isCygwin)
```

Comment:
`isWindows`, then?

Reply:
I wasn't sure about that.  Don't we also support msvc in some way?